app/src/main/java/com/shagalalab/constitution/ui/search/SearchResultFragment.kt

```diff
@@ -32,7 +32,12 @@ class SearchResultFragment : Fragment(R.layout.fragment_search_result) {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        tvResults.text = getString(R.string.results_for, safeArgs.query)
+        when (safeArgs.lang) {
+            Language.QQ.ordinal -> tvResults.text = getString(R.string.qq_results_for, safeArgs.query)
+            Language.RU.ordinal -> tvResults.text = getString(R.string.ru_results_for, safeArgs.query)
+            Language.UZ.ordinal -> tvResults.text = getString(R.string.uz_results_for, safeArgs.query)
+            Language.EN.ordinal -> tvResults.text = getString(R.string.en_results_for, safeArgs.query)
```

Comment:
Well, not 100% as I requested. As you can see, we have a code repetition here, we are assigning some string to `tvResults.text` 4 times. This repetition can be easily solved by moving the `tvResults.text = ` part out of the `when` statement.